src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -1863,6 +1869,9 @@
   <data name="ERR_InterfaceImplementedTwice1" xml:space="preserve">
     <value>Interface '{0}' can be implemented only once by this type.</value>
   </data>
+  <data name="ERR_InterfaceImplementedTwiceWithDifferentTupleNames" xml:space="preserve">
+    <value>Interface '{0}' can be implemented only once by this type, but already appears with different tuple element names, as '{1}'.</value>
+  </data>
```

Comment:
></data> [](start = 2, length = 7)

Usually there are corresponding changes in .Designer.vb file, but it doesn't look like the file is included into the PR.